design/README.md

```diff
@@ -0,0 +1,28 @@
+The budjen appliction needs to print a list of transactions in chronological order.
+
+Each class in the budjen application should one have one purpose (shown as =) and possess certain attributes (shown as -).
+
+Budjen class = run the application.
+- No fields.
+- One method called main, which will call the start method from the Application class.
+
+Application class = print a list of transactions in chronological order.
+- Four dependency fields called transactionData, orderer, transactionFormatter and logger.
+- One method called start that prints a list of transactions in chronological order.
+
+TransactionData class = provide the transaction data for the Application class.
+- One public method called getData.
+- One private method called makeTransaction to make a new transaction that sets the type, description, money and date.
```

Comment:
No need to really mention what private methods do. `private` basically means "it would be nice if there was a method for doing this, but this shouldn't be known outside of this class". Applies throughout.